extensions/tls-registry/pom.xml

```diff
@@ -16,6 +16,8 @@
     <packaging>pom</packaging>
     <modules>
         <module>deployment</module>
+        <module>deployment-spi</module>
```

Comment:
How did you draw the line between what goes in deployment and what goes in deployment-spi?

Reply:
The way I see it, `deployment` contains build items that are only supposed to tbe used by the extension itself, while the ones it "communicates" with other extensions should go to `deployment-spi`

Reply:
I think it's okay to not be too restrictive about this. Items should not carry any behavior.